spec/features/idv/hybrid_mobile/hybrid_mobile_spec.rb

```diff
@@ -474,12 +474,4 @@
   end
 
   it_behaves_like 'hybrid flow doc auth'
-
```

Comment:
this approach technically works ... it may be worth undoing the [shared_example approach implemented for strictly split doc auth testing](https://github.com/18F/identity-idp/pull/11263/commits/e60062a65621ba85ee559e3c4b56ce59a8463a81#diff-2bbe9e2925706d4490e1684caf66690089faeed98fbd48a19d067b91ede1746a) which is no longer need here .. should be a quick change ... happy to pair on it if it helps